docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -4,10 +4,10 @@
    "cell_type": "markdown",
```

Comment:
<p>device="gpu" is not necessary for brightness_contrast - it will be inferred from the input.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Done</p>